CODEOWNERS

```diff
@@ -49,12 +49,12 @@
 /doc/cprover-manual/contracts* @tautschnig @feliperodri @SaswatPadhi
 /src/goto-diff/ @tautschnig @peterschrammel
 /src/jsil/ @kroening @tautschnig
-/src/memory-analyzer/ @tautschnig @chrisr-diffblue
+/src/memory-analyzer/ @tautschnig @chrisr-diffblue @peterschrammel
 /jbmc/src/jbmc/ @peterschrammel @romainbrenguier
 /jbmc/src/janalyzer/ @peterschrammel @romainbrenguier
-/jbmc/src/jdiff/ @peterschrammel
+/jbmc/src/jdiff/ @peterschrammel @chrisr-diffblue
 /src/cpp/ @kroening @tautschnig @peterschrammel
-/src/solvers/smt2 @kroening @martin-cs @tautschnig @peterschrammel @allredj @romainbrenguier
```

Comment:
While at it, please remove my name from this one. I do not feel sufficiently competent/confident for SMT2.

Reply:
Done in pending commit.

Reply:
OK for me. I'm not active on SMT2 at the moment.